x/participationrewards/keeper/keeper.go

```diff
@@ -21,10 +21,10 @@ import (
 	"github.com/ingenuity-build/quicksilver/x/participationrewards/types"
 )
 
-// userAllocation is an internal keeper struct to track transient state for
+// UserAllocation is an internal keeper struct to track transient state for
 // rewards distribution. It contains the user address and the coins that are
 // allocated to it.
-type userAllocation struct {
+type UserAllocation struct {
 	Address string
 	Amount  math.Int
 }
```

Comment:
We should move this to` types` now that it is exported.

Reply:
second that

Reply:
done